lib/Service/AvatarService.php

```diff
@@ -148,6 +137,22 @@ private function getAvatarFolder(string $token): ISimpleFolder {
 		return $avatarFolder;
 	}
 
+	/**
+	 * https://github.com/sebdesign/cap-height -- for 500px height
+	 * Automated check: https://codepen.io/skjnldsv/pen/PydLBK/
+	 * Noto Sans cap-height is 0.715 and we want a 200px caps height size
+	 * (0.4 letter-to-total-height ratio, 500*0.4=200), so: 200/0.715 = 280px.
+	 * Since we start from the baseline (text-anchor) we need to
+	 * shift the y axis by 100px (half the caps height): 500/2+100=350
+	 *
+	 * Copied from @see \OC\Avatar\Avatar::$svgTemplate with only `{font}` being modified
+	 */
+	private string $svgTemplate = '<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+		<svg width="{size}" height="{size}" version="1.1" viewBox="0 0 500 500" xmlns="http://www.w3.org/2000/svg">
+			<rect width="100%" height="100%" fill="#{fill}"></rect>
+			<text x="50%" y="350" style="font-weight:normal;font-size:280px;font-family:{font};text-anchor:middle;fill:#{fgFill}">{letter}</text>
```

Comment:
```suggestion
			<text x="50%" y="350" style="font-weight:normal;font-size:240px;font-family:{font};text-anchor:middle;fill:#{fgFill}">{letter}</text>
```

Very nice, though some emojis almost touch the boundaries of the avatar circle. I would slightly scale them and give them some air.